src/datasource/DataSource.ts

```diff
@@ -352,6 +353,14 @@ export class SMDataSource extends DataSourceApi<SMQuery, SMOptions> {
       });
   }
 
+  async getTenantLimits(): Promise<ListTenantLimitsResponse> {
+    return firstValueFrom(
+      getBackendSrv().fetch({ method: 'GET', url: `${this.instanceSettings.url}/sm/tenant/limits` })
+    ).then((res: any) => {
+      return res.data;
+    });
+  }
```

Comment:
Just going to bump this comment in case you missed it. For all intents and purposes of how we use this function it is virtually the same as what you had except gets rid of what could be problematic explicit any types if `@grafana/runtime` was updated and changed their behaviour for any reason.

Reply:
Whoops missed this one